WORKSPACE

```diff
@@ -26,32 +26,34 @@ http_archive(
     urls = ["https://github.com/bazelbuild/rules_docker/archive/v0.7.0.tar.gz"],
 )
 
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
```

Comment:
All `WORKSPACE`, `BUILD.bazel` and `*.bzl` files are formatted using [Buildifier](https://github.com/bazelbuild/buildtools/tree/master/buildifier). Could you please rerun this? Thanks!

Reply:
For `BUILD.bazel` files it's good to know that a `bazel run //:gazelle` already reformats those.